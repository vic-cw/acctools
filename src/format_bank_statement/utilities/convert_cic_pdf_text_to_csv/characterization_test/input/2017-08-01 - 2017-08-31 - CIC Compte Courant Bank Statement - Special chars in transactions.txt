                                                                       Crédit Industriel et Commercial

                                                                                                                   CIC PARIS SAINT MARTIN

                                                                                                         224 RUE SAINT MARTIN 75010 PARIS
                                                   ¤     u0820 81 23 45 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                                                                                              12345@cic.fr                                      BIC : CMCIFRPP

                                                                                                                                                                                                                45678

                                                                                                                                                                                                                12345

                                                                                                                          M  MARC   JOHNSON

                                                                                                                          1  RUE      DE   TRUC

                                                                                                                          93100       NEW YORK

...........................................................................................................................V..O...T.R..E...C..O..N..S..E..I.L..L..E..R..:..M...S..é..b..a.s..t.i.e.n...T..R..O..U..D.U.C....................

                                             RELEVE ET INFORMATIONS BANCAIRES                                                                                                                                Agence 12345

                                             31 août 2017

                                                   C/C CONTRAT PERSONNEL GLOBAL N° 00012345678 en euros (GD)

                                                   TITULAIRE(S) : M MARC JOHNSON

                                             Date        Date valeur                    Opération                                                                                               Débit EUROS  Crédit EUROS

                                                                                        SOLDE CREDITEUR AU 31/07/2017                                                                                        1,00                                                                                        Pour tout appel en vue d'obtenir la bonne exécution d'un contrat ou le traitement

                                             01/08/2017  01/08/2017                    ONE LINE EXPENSE                                                                                         0,01

                                             02/08/2017  02/08/2017                    PAIEMENT CB    2907 PARIS0123456/                                                                        24,79

                                                                                       VENDEUR SARL    CARTE 00123456

                                             07/08/2017  07/08/2017                    PAIEMENT CB    0508 3,99     USD                                                                          3,40

                                             (GE) : protégé par la Garantie de l'Etat                                                                                                                        <<Suite au verso>>

                                             (GD) : protégé par la Garantie des Dépôts

                                             Remarque : Le livret réglementé relève de la garantie d'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des dépôts.

                                             www.garantiedesdepots.fr

                                                                                                                                                                                                             Page 1

                                               CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

                                         6,  avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

                                                                                        M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - www.lemediateur-cic.fr

                                                                       Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
                                         RELEVE ET INFORMATIONS BANCAIRES                                                                                                                                Agence 12345

                                         31 août 2017

                                               C/C CONTRAT PERSONNEL GLOBAL N° 00012345678                                en  euros  (GD)

                                               TITULAIRE(S) : M MARC JOHNSON

                                         Date        Date valeur                    Opération                                                                                               Débit EUROS  Crédit EUROS

                                                                                   C1 - TRUC CHOSE CARTE 00123456

                                         16/08/2017  16/08/2017                    RETRAIT DAB 1408 EVRY                                                                                    40,00

                                         (GE) : protégé par la Garantie de l'Etat

                                         (GD) : protégé par la Garantie des Dépôts

                                         Remarque : Le livret réglementé relève de la garantie d'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des dépôts.

                                         www.garantiedesdepots.fr

                                                                                                                                                                                                         Page 2
                                                                       Crédit Industriel et Commercial

                                                                                                                CIC PARIS SAINT MARTIN

                                                   ¤                                                    224 RUE SAINT MARTIN 75010 PARIS
                                                         u0820 81 23 45 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                                                                                              12345@cic.fr                                             BIC :   CMCIFRPP

                                             RELEVE ET INFORMATIONS BANCAIRES                                                                                                                                  Agence 12345

                                             31 août 2017

                                                   C/C CONTRAT PERSONNEL GLOBAL N° 00012345678                                     en  euros  (GD)

                                                   TITULAIRE(S) : M MARC JOHNSON

                                             Date        Date valeur                    Opération                                                                                               Débit  EUROS   Crédit EUROS

                                                                                       PLACE EXT.   CARTE 00123456

                                             22/08/2017  22/08/2017                    PAIEMENT CB  2108 PROUT                                                                                         80,00

KV.20170901.012345.6789.1011.1213 X 0 2                                                SOMETHING " CARTE 00123456

                                             23/08/2017  23/08/2017                    PAIEMENT CB  2108 REPROUT                                                                                       3,00

                                                                                       SECOND LINE CARTE *00123456


                                             (GE) : protégé par la Garantie de l'Etat                                                                                                                          <<Suite au verso>>

                                             (GD) : protégé par la Garantie des Dépôts

                                             Remarque : Le livret réglementé relève de la garantie d'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des dépôts.

                                             www.garantiedesdepots.fr

                                                                                                                                                                                                               Page 3

                                               CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

                                         6,  avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

                                                                                        M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - www.lemediateur-cic.fr

                                                                       Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
                                         RELEVE ET INFORMATIONS BANCAIRES                                                                                                                                   Agence 12345

                                         31 août 2017

                                               C/C CONTRAT PERSONNEL GLOBAL N° 00012345678                          en  euros  (GD)

                                               TITULAIRE(S) : M MARC JOHNSON

                                         Date        Date valeur                    Opération                                                                                               Débit  EUROS    Crédit EUROS

                                         28/08/2017  28/08/2017                    PAIEMENT CB   1908 MONTROUGE                                                                                    30,71

                                                                                   APOSTROPHE'      CARTE 00123456

                                         30/08/2017  30/08/2017                    PAIEMENT CB   2808 LAST                                                                                         50,05

                                                                                   BUT NO LEAST  CARTE 00123456

                                         Réf : 008                                  SOLDE CREDITEUR AU 31/08/2017                                                                                           0,01

                                         QXBAN : AB01 CMCI FRPP XXX CDE2 FGHI 3JKL MNOP 4QR                             IBAN : FR76 0123 4567 8910                                                          1112 1314 151

                                           SITUATION DE VOS CREDITS en euros au 31 août 2017

                                         Numéro                                     Libellé du compte                          Montant emprunté                                                             Solde (*)

                                         00012345678               ALLURE LIBRE                                         EUR                                                                        +0,00  +0,00

                                         (*) Solde correspondant à l'amortissement normal du prêt

                                         AUTORISATIONS DE DECOUVERTS

                                         N° de compte                               Libellé                                                                                                                 Montant euros

                                         00012345678               DECOUVERT C. PERS. GLOBAL TAEG 15,12% AU 27/07/2017                                                                                      1.000.000.000,00

KV.20170901.012345.6789.1011.1213 X 0 V  TOTAL MENSUEL DES FRAIS DEBITES SUR PRODUITS ET SERVICES BANCAIRES

                                         N° de compte                               Libellé                                                                                                                 Montant euros

                                         00012345678               TOTAL DES FRAIS EN JUL 2017                                                                                                              1,62

                                         (GE) : protégé par la Garantie de l'Etat

                                         (GD) : protégé par la Garantie des Dépôts

                                         Remarque : Le livret réglementé relève de la garantie d'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des dépôts.

                                         www.garantiedesdepots.fr

                                                                                                                                                                                                            Page 4
