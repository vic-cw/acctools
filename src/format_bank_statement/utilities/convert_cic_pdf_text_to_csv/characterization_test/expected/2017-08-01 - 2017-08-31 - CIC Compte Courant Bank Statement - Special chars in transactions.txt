2017-08-01,2017-08-01,"ONE LINE EXPENSE","-0,01"
2017-08-02,2017-08-02,"PAIEMENT CB    2907 PARIS0123456/ VENDEUR SARL    CARTE 00123456","-24,79"
2017-08-07,2017-08-07,"PAIEMENT CB    0508 3,99     USD C1 - TRUC CHOSE CARTE 00123456","-3,40"
2017-08-16,2017-08-16,"RETRAIT DAB 1408 EVRY PLACE EXT.   CARTE 00123456","-40,00"
2017-08-22,2017-08-22,"PAIEMENT CB  2108 PROUT SOMETHING "" CARTE 00123456","-80,00"
2017-08-23,2017-08-23,"PAIEMENT CB  2108 REPROUT SECOND LINE CARTE *00123456","-3,00"
2017-08-28,2017-08-28,"PAIEMENT CB   1908 MONTROUGE APOSTROPHE'      CARTE 00123456","-30,71"
2017-08-30,2017-08-30,"PAIEMENT CB   2808 LAST BUT NO LEAST  CARTE 00123456","-50,05"
